.github/workflows/update_sdk_methods.py

```diff
@@ -1589,20 +1599,25 @@ def write_markdown(type, names, methods):
                             ## If the method has a code sample, print it here:
                             if 'code_sample' in methods['flutter'][type][resource][flutter_method_name]:
 
-                                output_file.write('``` ' + code_fence_fmt['flutter'] + ' {class="line-numbers linkable-line-numbers"}\n')
+                                output_file.write('```' + code_fence_fmt['flutter'] + ' {class="line-numbers linkable-line-numbers"}\n')
                                 output_file.write(methods['flutter'][type][resource][flutter_method_name]['code_sample'])
                                 output_file.write('```\n\n')
 
                             ## If we detected an 'after' method override file earlier, write it out here:
                             if has_after_override:
-                                output_file.write('\n')
 
                                 for line in open(method_override_file_path, 'r', encoding='utf-8'):
                                     output_file.write(line)
 
+                                output_file.write('\n')
+
                             output_file.write("{{% /tab %}}\n")
                             output_file.write("{{< /tabs >}}\n")
 
+                    ## After this loop, we will be working with additional methods appended to the same {resource}.md include file.
+                    ## This switch tells us at the start of the loop for this same resource that we should double-newline the next
+                    ## proto encountered:
+                    is_first_method_in_this_resource = 0
```

Comment:
This works but you're kind of using an integer as a boolean here
```suggestion
                    is_first_method_in_this_resource = False
```

Reply:
You're right, this is cleaner, thanks!